sdk/sdk/Models/dao/OBAModelDAOUserPreferencesImpl.m

```diff
@@ -243,8 +235,6 @@ - (void) writeSetRegionAutomatically:(BOOL)setRegionAutomatically {
     NSUserDefaults * user = [NSUserDefaults standardUserDefaults];
     NSNumber * v = @(setRegionAutomatically);
     [user setObject:v forKey:kSetRegionAutomaticallyKey];
-
-    [[GAI sharedInstance].defaultTracker set:[GAIFields customDimensionForIndex:2] value:(setRegionAutomatically ? @"YES" : @"NO")];
```

Comment:
Did not want the shared library to be coupled to GA.  
A few reasons for not wanting that, separation of concerns, future extensibility, etc.

As a general rule, trying to maintain single responsibility for a particular entity has long term maintainability benefits.